<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWidget</class>
 <widget class="QWidget" name="CalculatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="1" column="6">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radiansBtn">
       <property name="text">
        <string>Radians</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="degreesBtn">
       <property name="text">
        <string>Degrees</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="equation">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyChangesBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="7">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="8">
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <widget class="QPushButton" name="divideBtn">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QPushButton" name="multiplyBtn">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="floorBtn">
       <property name="text">
        <string>floor</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QPushButton" name="subtractBtn">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="ceilBtn">
       <property name="text">
        <string>ceil</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="cosBtn">
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="tanBtn">
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="acosBtn">
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="sinBtn">
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="asinBtn">
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5" colspan="2">
      <widget class="QPushButton" name="zeroBtn">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="atanBtn">
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="fourBtn">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="sixBtn">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="eightBtn">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QPushButton" name="decimalBtn">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="twoBtn">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="sevenBtn">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="threeBtn">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="nineBtn">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="fiveBtn">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="oneBtn">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="lnBtn">
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="log10Btn">
       <property name="text">
        <string>log10</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="logBtn">
       <property name="text">
        <string>log(b,x)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="sqrtBtn">
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5" colspan="2">
      <widget class="QToolButton" name="scalarsBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Scalars</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="7" colspan="2">
      <widget class="QToolButton" name="vectorsBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Vectors</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="rootBtn">
       <property name="text">
        <string>root(r, ind)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="powBtn">
       <property name="text">
        <string>x^y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="backspaceBtn">
       <property name="text">
        <string>Backspace</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="leftBraceBtn">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="rightBraceBtn">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="commaBtn">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="absBtn">
       <property name="text">
        <string>abs</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="expBtn">
       <property name="text">
        <string>e^y</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="piBtn">
       <property name="text">
        <string>pi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
